includes/islandora_book_batch.inc

```diff
@@ -466,15 +466,30 @@ class IslandoraBookPageBatchObject extends IslandoraFlatBatchObject {
    */
   public function addRelationships() {
     module_load_include('inc', 'islandora_paged_content', 'includes/utilities');
-    // Add relationship to collection.
+    // Add relationship to parent.
```

Comment:
This is not really the parent. This relationships are the ones for each child (page)

Reply:
Depends how you read it. We're adding the relationship on the child to the parent. "Add relationship to parent; I'm not talking about actual relationship (isPageOf etc.).

Reply:
ohh.. well, you could interpret it like that, i give you that! but it requires some extra reading (or i'm slow), and well, its also not 100% true since you are adding also literals to the child that are not related to the parent in any way (section, page number, etc). But ok, i don't have really a strong opinion here, does not change the way the code runs 😄

Reply:
neither is "collection" ;)

Reply:
Why not either change to "Add relationships/literals" or move this comment above line 471 and move line 475 up there as well. Then you'd have
```
// Add relationship to parent.
islandora_paged_content_set_relationship($rels_ext, ISLANDORA_RELS_EXT_URI, 'isPageOf', $this->parentId);
islandora_paged_content_set_relationship($rels_ext, FEDORA_RELS_EXT_URI, 'isMemberOf', $this->parentId);
```